src/auth.rs

```diff
@@ -188,6 +188,7 @@ pub enum KVAction {
     },
     List {
         orbit: Orbit,
+        prefix: String,
```

Comment:
Should that be an `Option`?

Reply:
There should always be a `/` after the service: `kepler:ens:example.eth://default/kv/#list`, so in this case the prefix would be `""`.